recipe/recipe.yaml

```diff
@@ -0,0 +1,86 @@
+schema_version: 1
+
+context:
+  name: zfit
+  version: 0.25.0
+  python_min: 3.9
+
+package:
+  name: ${{ name|lower }}
+  version: ${{ version }}
+
+source:
+  url: https://pypi.org/packages/source/${{ name[0] }}/${{ name }}/${{ name }}-${{ version }}.tar.gz
+  sha256: ac5a92bc284094eae55dd9afe1fe2c8f3f67a402dfc7a8ad6087a9ea29ff2b41
+
+build:
+  number: 1
+  noarch: python
+  script: ${{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python 3.9.*
```

Comment:
```suggestion
    - python ${{ python_min }}.*
```

Reply:
Thanks! I've had some issues with the conflict resolution, therefore created this new branch